CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ### Fixed
 - Fixed `'??' has non-optional type` warning #207
 - Fixed incorrect replacements in server variables #209
+- Fixed missing support for multiple authentication types per operation #214 #222 @malteburkert
+- Fixed nullable references not being generated as optionals #216 @alephao
```

Comment:
You’ll need to rebase or merge master in. PR #216 is in 4.3.1 that has since been released